web/content/docs/tools/workflows/create-a-simple-parallel-model/partmesh.pandoc

```diff
@@ -0,0 +1,17 @@
+
+# OpenGeosys  parallel simulation model preparation command tool:  partmesh
+
+**partmesh**  is a command tool to generate parted mesh from vtu files. It will run **mpmetis** to get a parted index for nodes, and then write parted meshes&nodes&elements&properties files as binary/ASCII format. Here introduce the options and workflow of this command tool:
+
+option | detail
+-------- | -----
+-i | define the input mesh file, most time is a vtu file
+-o | define the output directory
+-n/np | number of partition
+-s/ogs2metis | switch on/off conversion from vtu to metis input file
+-m/exe_metis | call mpmetis inside the program
+-q/lh_elements | partition with mixed order elements
+-a/ascii | enable ascii format files output 
+--| other mesh files, such as mesh files of boundaries
+
+![Workflow of partmesh command tool](partmesh.png)
```

Comment:
The link to the png file fails. You can use  ``{{< img src="../partmesh.png" >}}``.

Reply:
sorry,just find this msg, just uploaded to repo.